src/elements/hx-alert/index.spec.js

```diff
@@ -7,32 +7,70 @@ import { fixture, expect } from '@open-wc/testing';
  *
  */
 describe('<hx-alert> tests', () => {
-
     const template = '<hx-alert>';
 
-    it('render the Shadow Root mode open', async () => {
-        const el = /** @type { HXAlertElement } */ await fixture(template);
-        const mode = el.shadowRoot.mode;
+    describe('test instantiate element', () => {
```

Comment:
@BY00565233 @lalithkarikelli 

To assist with the new tests we are writing for each component, I'm including a draft for `<hx-alert>`.  To help with breaking down what is needed to be implemented, I first add empty `describe` blocks:

<img width="602" alt="Screen Shot 2020-05-08 at 2 56 39 PM" src="https://user-images.githubusercontent.com/10120600/81444273-7c6ffa00-913c-11ea-87b2-01b02523e3e5.png">

Then you can go back and add tests to each `describe` block.